recipe/meta.yaml

```diff
@@ -36,7 +36,7 @@ outputs:
         - cmake
       host:
         - ninja
-        - python
+        - python >=3.10
```

Comment:
python <3.9 already removed from https://github.com/conda-forge/muscat-split-feedstock/blob/main/.ci_support/linux_64_.yaml

dont need the ">=3.10"

also you can remove all the py == 39 and other python 3.9 selectors